packages/augur-ui/src/modules/app/components/inner-nav/markets-list-filters.tsx

```diff
@@ -55,7 +55,7 @@ export default class MarketsListFilters extends React.Component<
         selectedFee: filterOptions.maxFee || this.props.maxFee,
         selectedSpread: filterOptions.spread || this.props.maxLiquiditySpread,
         showInvalidDefault:
-          filterOptions.showInvalid || this.props.showInvalid,
+          filterOptions.showInvalid || this.props.includeInvalidMarkets,
```

Comment:
why is there two properties here to set `shwInvalidDefault` ?

Reply:
thanks for the catch Tom!

Reply:
@bthaile correction.

filterOptions is coming from params. So if we have the param on the URL we use it, otherwise we fallback to includeInvalidMarkets (which is our reducer state).

I updated the var name to be more clear.